superdesk/metadata/item.py

```diff
@@ -308,6 +308,16 @@
         'type': 'string'
     },
 
+    # Desk and Stage Details
```

Comment:
This already exists in `common.py.item_schema()`. This will be overwritten by archive resource as we do `metadata_schema.update(item_schema())`.https://github.com/superdesk/superdesk/blob/master/server/apps/archive/common.py#L385